src/main/java/net/sf/jabref/exporter/FieldFormatterCleanups.java

```diff
@@ -85,7 +85,7 @@ public int hashCode() {
         return Objects.hash(actions, enabled);
     }
 
-    private static List<FieldFormatterCleanup> parse(String formatterString) {
+    public static List<FieldFormatterCleanup> parse(String formatterString) {
```

Comment:
Ok, I think your solution is more readable. Ignore my comment and keep the parse method public.